docs/source/modules/event.rst

```diff
@@ -0,0 +1,51 @@
+.. _event-module:
+
+Event Module
+============
+
+.. contents::
+    :depth: 2
+
+Description
+-----------
+
+The Event module allows to create listener and assign action to event. The module is using :code:`MotechEvent` to provide
+event with specific data that will be sent by Motech Scheduler when a scheduled event is fired.
+The module is using ActiveMQ to support the event execution and callback. It is possible to register your own callback
+function using :code:`EventCallbackService`.
```

Comment:
An example of an implemented callback service would be useful, together with the way of firing an event so the callback is hit